src/commands/deploy.ts

```diff
@@ -489,6 +263,7 @@ export default class Deploy extends DeployCommand {
   }
 
   async run(): Promise<void> {
+    this.log(JSON.stringify(this.argv));
```

Comment:
debug code?

Reply:
No, clearly this is very important code! (Already removed thanks for the catch!)